apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -239,120 +331,33 @@ onDocumentReady(() => {
         formatter: timeRemainingLimitFormatter,
         class: 'text-center align-middle',
         switchable: true,
-        searchFormatter: false,
+      },
+      {
+        field: 'client_fingerprint_id_change_count',
+        title: html` Fingerprint Changes
+          <button
+            class="btn btn-xs"
+            type="button"
+            title="Show fingerprint changes help"
+            data-toggle="modal"
+            data-target="#fingerprint-changes-help"
+          >
+            <i class="bi-question-circle-fill" aria-hidden="true"></i>
+          </button>`,
+        class: 'text-center align-middle',
+        // Hidden for groupwork by default, as it is not as relevant in that context
+        visible: !assessmentGroupWork,
```

Comment:
Minor change: I added the fingerprint column unconditionally, but made it hidden by default for group assessments.

Reply:
Does this mean it can be toggled on via the "Columns" selector? I don't think this is a problem, just curious.

Reply:
Indeed.